src/apolloServer.ts

```diff
@@ -0,0 +1,48 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { AppDataSource } from './data-source';
+import { ApolloServer } from 'apollo-server';
+import { ApolloServerPluginLandingPageLocalDefault } from 'apollo-server-core';
+import { User } from './entity/User';
+
+interface UserInput {
+  id?: number;
+  name: string;
+  email: string;
+  password: string;
+  birthdate: string;
+}
+
+async function initApolloServer() {
+  const resolvers = {
+    Query: {
+      users: () => AppDataSource.manager.getRepository('user').createQueryBuilder('user').getMany(),
+    },
+    Mutation: {
+      createUser: async (parent: any, args: UserInput) => {
+        const newUser = new User();
+        newUser.name = args.name;
+        newUser.email = args.email;
+        newUser.password = args.password;
+        newUser.birthdate = args.birthdate;
```

Comment:
```suggestion
        Object.assign(newUser, {
          name: args.name,
          email: args.email,
          password: args.password,
          birthDate: args.birthDate,
        });
```

Assim não repete tanto os campos.

Reply:
Feito